internal/controller/standalone_pgadmin/controller.go

```diff
@@ -81,9 +81,8 @@ func (r *PGAdminReconciler) watchPostgresClusters() handler.Funcs {
 	}
 }
 
-//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins,verbs={get,list,watch,create,update,patch,delete}
-//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins/status,verbs={get,update,patch}
-//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins/finalizers,verbs={update}
+//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins,verbs={get}
+//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins/status,verbs={patch}
```

Comment:
⛏️ I like the quotes in the other one.
```suggestion
//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgadmins",verbs={get}
//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgadmins/status",verbs={patch}
```

Reply:
Quotes is the pattern elsewhere, so 👍 to that

Reply:
:+1: Updated.